### ISSUE TITLE
Missing _write_ts field

### ISSUE THREAD

**Issue by aousterh** [brimsec/zq-sample-data#28]
The zed-sample-data (commit [54c1e8a](https://github.com/brimdata/zed-sample-data/commit/54c1e8ab8c1e2bbd19820f3441b1d01a08c177b7)) seems to be inconsistent about whether records contain the `_write_ts` field or not. The field seems to be present in the zeek-ndjson format of the data, but not in any of the other formats. For example:
```
$ zq -Z 'has(_write_ts) | count()' zeek-ndjson/*
{
    count: 1462078 (uint64)
} (=0)
$ zq -Z 'has(_write_ts) | count()' zng/*
$
```
If I shape the zeek-ndjson data myself, the resulting data does have the `_write_ts` field in it:
```
$ zq -I shaper.zed zeek-ndjson/* | zq -Z 'has(_write_ts) | count()' -
{
    count: 1462078 (uint64)
} (=0)
```
So it seems to be an inconsistency between the different formats of the sample data, rather than an issue with zq or shaper.zed.

**Comment by philrz** [comment]
HI @aousterh. Yes, your observation is certainly correct. It's one of those things that's a bug or a feature depending on how you look at it. :smile:   As the [README](https://github.com/brimdata/zed-sample-data/blob/main/README.md) gets into, the sample data was generated by running the same pcap through a particular Zeek version with the [JSON Streaming Logs](https://github.com/corelight/json-streaming-logs) responsible for outputting the NDJSON and the system otherwise at its defaults.  In this regard, the `zeek-default` and `zeek-ndjson` ones are kind of "special" because they both were directly output by Zeek, whereas the other ones like `zng`, `zson`, etc. were after running the `zeek-default` ones through `zq` to generate the alternate formats. It just so happens that the `_write_ts` field is included in that NDJSON they output. It's been a while since I've heard a detailed explanation for why it's even there, but I think it might have to do with the fact that users often stream NDJSON over the network (e.g., via Kafka) rather than writing it directly to the filesystem, so having the timestamp of when the record was generated helps to track network latency issues.

Back to bug vs. feature, at the time we first generated this data set Zeek, was still very much the first (and almost only) kind of input data we were concerned with, so keeping the sample data as a pure representation of Zeek output defaults was justifiable in terms of establishing that Brim & the Zed tools were prepped for out-of-the-box use with Zeek. For instance, having the NDJSON variant was largely motivated by wanting to show that the application of a Zed shaper (or, at the time, the prior approach we had to applying rich data types to NDJSON) would effectively bring the data back to the level of richness that's in the TSV equivalent, with the exception of that `_write_ts` field being present because, well, that's just part of the data. OTOH, if you're doing an apples-to-apples comparison of performance when reading the various formats or something like that (which I suspect you are), I respect that the presence of an additional field kind of disturbs the apples-to-apples nature of it. In that case I might just recommend making your own local adjusted version of the NDJSON one with that field dropped. In full disclosure, this Zeek sample data is expected to become less prominently featured in the project over time, as I've got branches for a new sample data set that's more "generic" in nature, since we want to emphasize that Zed is ready for any & all data, not just Zeek/Suricata or other network security logs. Since we have some history/automation with this particular Zeek data I plan to keep it around in the repo, but I plan to tuck it deeper in the directory hierarchy and mention it more of an aside in the top-level README.
